<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListItemWidget</class>
 <widget class="QWidget" name="ListItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>740</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1
2
3</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>20</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重新
下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenDir">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开
目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开
文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
